content/pl/releases/download.md

```diff
@@ -0,0 +1,51 @@
+
+## Obrazy kontenerów
+
+Wszystkie obrazy kontenerowe umieszczane są w rejestrze `registry.k8s.io`.
+
+{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
+
+Dla wersji Kubernetesa {{< param "version">}} następujące obrazy kontenerów opatrzone są podpisem [cosign](https://github.com/sigstore/cosign):
+
+| Obraz kontenera                                                           | Wspierana architektura                                                                   |
+| ------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| registry.k8s.io/kube-apiserver:v{{< skew currentPatchVersion >}}          | amd64, arm, arm64, ppc64le, s390x |
+| registry.k8s.io/kube-controller-manager:v{{< skew currentPatchVersion >}} | amd64, arm, arm64, ppc64le, s390x |
+| registry.k8s.io/kube-proxy:v{{< skew currentPatchVersion >}}              | amd64, arm, arm64, ppc64le, s390x |
+| registry.k8s.io/kube-scheduler:v{{< skew currentPatchVersion >}}          | amd64, arm, arm64, ppc64le, s390x |
+| registry.k8s.io/conformance:v{{< skew currentPatchVersion >}}             | amd64, arm, arm64, ppc64le, s390x |
+
+Obrazy kontenerów Kubernetesa obsługują rozmaite architektury sprzętowe, ich wyboru powinno zaś dokonać środowisko uruchomieniowe w zależności od wybranej platformy. Istnieje też możliwość pobrania obrazu kontenera dla konkretnej architektury poprzez dodanie przyrostka do jego nazwy, np. `registry.k8s.io/kube-apiserver-arm64:v{{< skew currentPatchVersion >}}`. Wszystkie te warianty obrazów Kubernetesa są podpisane w taki sam sposób jak w przypadku listy manifestów wieloarchitekturowych. 
+
+Wydawcy Kubernetesa publikują listę podpisanych obrazów kontenerowych w formacie [SPDX 2.3](https://spdx.dev/specifications/). Możesz ją pobrać wpisując w powłoce: 
+
+```shell
+curl -Ls "https://sbom.k8s.io/$(curl -Ls https://dl.k8s.io/release/stable.txt)/release" | grep "SPDXID: SPDXRef-Package-registry.k8s.io" |  grep -v sha256 | cut -d- -f3- | sed 's/-/\//' | sed 's/-v1/:v1/'
+```
+
+Dla wersji {{< skew currentVersion >}} Kubernetesa jedynym typem artefaktu kodu, którego integralność możesz zweryfikować, jest obraz kontenera (korzystając z eksperymentalnej opcji podpisu). 
+
+By ręcznie zweryfikować podpisane obrazy kontenerów głównych komponentów Kubernetesa, zobacz [Zweryfikuj podpisane obrazy kontenerów](/docs/tasks/administer-cluster/verify-signed-artifacts).
+
+## Pliki binarne
+
+Odnośniki do plików binarnych z komponentami Kubernetesa (i ich sumy kontrolne) znajdziesz w [CHANGELOGU](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG).
+
+Możesz też skorzystać ze strony [downloadkubernetes.com](https://www.downloadkubernetes.com/), gdzie szukane komponenty możesz filtrować w zależności od wersji i architektury sprzętowej.
```

Comment:
```suggestion
{{< release-binaries >}}
```
(and then verify that you've added all the relevant strings into [`data/i18n/pl/pl.toml`](https://github.com/kubernetes/website/blob/main/data/i18n/pl/pl.toml))

Reply:
Hello @sftim! Thank you for catching this difference; I had clearly overlooked the change in the English version. I've made the correction and also included an updated and cleaned Polish version of the i18n file (/data/i18n/pl/pl.toml). @mfilocha, please take a look as well! Very best, Adam

Reply:
Are you happy with https://deploy-preview-43952--kubernetes-io-main-staging.netlify.app/pl/releases/download/#pliki-binarne? I think there might be some strings missing.

Reply:
This should change with the new data/i118n/pl/pl.toml file pushed which will happen shortly. I just need to do some cleaning.

Reply:
(still looks like missing strings to me - look for English text)

Reply:
Thanks for patience. With this addition github.com/kubernetes/website/pull/44054 the release-binaries part should be fine. Please check in your spare time.